orcid-core/src/main/resources/i18n/messages_en.properties

```diff
@@ -1902,6 +1902,7 @@ privacy_preferences.error_updating_visibility=Unable to update visibility prefer
 public-layout.about=About
 public-layout.about_orcid=About ORCID
 public-layout.account_setting=Account Settings
+public-layout.trusted_individuals=Trusted Individuals
```

Comment:
@leomendoza123 Can you add manually the properties for the testing languages XX, LR and RL. We don't have the GitHub actions that generate those properties like in orcid-angular